luma/core/sprite_system.py

```diff
@@ -66,7 +67,8 @@ class spritesheet(object):
     Loosely based on http://www.createjs.com/docs/easeljs/classes/SpriteSheet.html
     """
     def __init__(self, image, frames, animations):
-        self.image = Image.open(image)
+        self._img = open(image, 'rb')
+        self.image = Image.open(self._img)
```

Comment:
this works (tested), and doesn't rely on the consumer to call close:

```python
    def __init__(self, image, frames, animations):
        with open(image, "rb") as fp:
            self.image = Image.open(fp)
            self.image.load()

        ...
```

presumably it wont trigger the warning